Makefile

```diff
@@ -1,5 +1,13 @@
+all: clean lint fmt test
+
 clean:
 	rm -fr htmlcov .mypy_cache .pytest_cache .ruff_cache .coverage coverage.xml
+	hatch env remove unit
+
+install-dev:
+	pip install hatch
+	hatch env create
+	hatch run pip install -e '.[test]'
```

Comment:
Can you output the path to python interpreter here?

Reply:
as in `which python` ?

Reply:
I think Serge means this:
```
hatch run python -c "import sys; print(sys.executable)"
```